docs/immortal-empires/norsca.md

```diff
@@ -0,0 +1,30 @@
+# Norsca
+
+## Wulfrik the Wanderer - World Walkers
+
+* **Route I - Ragnarok**
+  * The End Times are nigh. Wulfrik has long cursed the gods for his predicament, but now realises he must pledge himself to one of them if he is to survive the coming apocalypse.
+  * Attain level 3 allegiance with a Chaos God.
+   * Defeat all of the Challengers sent against you.
+  * **Reward**:
+	* _Seafang Empowered_: Campaign movement range +30% (factionwide), enables teleportation for Wulfrik from port cities he owns (possible?)
+_Reward flavour text:_ Seafang's figurehead quickly absorbed the blood of the slain challengers that were smeared upon it. The ship now glows with eldritch power like never before.
+
+* **Route II - The World Walker**
+  * After a drunken boast made in sight of the Dark Gods, Wulfrik was cursed to forever wander the earth and seek out the fiercest challengers and most monstrous creatures and slay them in single combat.
+  * Occupy at least 5 of the following settlements:
+	* Couronne, Miragliano, Karaz-a-Karak, Altdorf, Castle Drakenhof, Black Crag, Kislev, Naggarond, Lothern, Hexoatl, Skavenblight, Khemri, The Awakening
+  * **Reward**:
+	* _The Eternal Challenger_: Enemy leadership -%10, enemy units near Wulfrik periodically Rampage (attacks the closest enemy and ignores any orders given).
+_Reward flavour text:_ With his Gift of Tongues Wulfrik curses foes and goads them into recklessness. Whether he's besmirching a Dwarf's honour, questioning a Bretonnian's lineage or humiliating a haughty Elf - none can ignore his insults for long.
+
+* **Route III - Where's My Palico?**
+  * Wulfrik leads his World Walkers across the world in search of mighty beasts to slay. Though the rewards are great, so too are the risks.
+  * Complete 6 Monster Hunts
+  * The Norse adventurer Losteriksson was the first Norscan to land on the coast of Lustria and live to tell the tale. The settlement he founded still stands to this day, though it is under dire threat from the Lizardmen of the jungle.
+  * Occupy Skeggi
+  * **Reward**:
+	* _Monster Hunter_: Primal Instincts (factionwide), Bonus vs Large +6 (factionwide)

```

Comment:
Both rewards are broken as fuck.

Reply:
Markus gets Primal Instincts for all **characters** and Bonus vs Large: +6 **vs Lizardmen** as two different rewards. What is proposed here is to mix them into one, give it to all units and against all enemies. Just any of **Primal Instincts (factionwide)** or **Bonus vs Large +6 (factionwide)** is already broken enough to mess the rest of the campaign.

Reply:
How bout primal instincts for all characters and bonus vs large only to the spearmen?

Reply:
I can accept Primal Insticts and Bonus vs Large: +6 for Marauder Hunters and Marauder Spearmen. Give them a reason to use cheap tier units. Feel free to add Bonus vs Large to their missiles too.